### ISSUE TITLE
NETSDK 1086 warning when referencing 4.3.1 and UseWindowsForms

### ISSUE THREAD

**Issue by jeremyVignelles** [dotnet/reactive#1112]
Hi,
When referencing System.Reactive in a WPF project using the `Microsoft.NET.Sdk.WindowsDesktop` SDK, both `UseWindowsForms` and `UseWPF`, and .NET core 3.1, I have this warning:

```
1>C:\Program Files\dotnet\sdk\3.1.100\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.Sdk.FrameworkReferenceResolution.targets(39,5): warning NETSDK1086: A FrameworkReference for 'Microsoft.WindowsDesktop.App' was included in the project. This is implicitly referenced by the .NET SDK and you do not typically need to reference it from your project. For more information, see https://aka.ms/sdkimplicitrefs
```

Library version : 4.3.1

Tried with both VS2019 and `dotnet build`.

You can find an example here : https://github.com/jeremyVignelles/repro-dotnet-reactive

**Comment by jeremyVignelles** [comment]
Any update ?

**Comment by jeremyVignelles** [comment]
Ping?
Is there anything else I need to do to help with that bug?

**Comment by ltrzesniewski** [comment]
This change was introduced in #1007 to fix #998.

@clairernovotny I'm not quite sure what's the best way to fix this, do you think that changing the condition to the following would be a good idea?

```
'$(UseWindowsRxVersion)' == 'true' and !@(FrameworkReference->AnyHaveMetadataValue('Identity', 'Microsoft.WindowsDesktop.App'))
```

**Comment by jeremyVignelles** [comment]
Thanks a lot @ltrzesniewski ! 🎉
